datareduction/datareducer.cpp

```diff
@@ -927,6 +927,41 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
 	 outputReducer->addMetadata(outputReducer->size()-1,"","","\\mathrm{Mesh data}$","");
          continue;
       }
+      if(lowercase == "vg_drho") {
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_drho",CellParams::D_RHO,1));
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\frac{\\Delta \\rho}{\\hat{rho}}$","");
+         continue;
+      }
+      if(lowercase == "vg_du") {
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_du",CellParams::D_U,1));
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\frac{\\Delta U_1}{\\hat{U}_1}$","");
+         continue;
+      }
+      if(lowercase == "vg_dpsq") {
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_dpsq",CellParams::D_PSQ,1));
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\frac{(\\Delta P)^2}{2 \\rho \\hat{U}_1}$","");
+         continue;
+      }
+      if(lowercase == "vg_dbsq") {
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_dbsq",CellParams::D_BSQ,1));
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\frac{(\\Delta B_1)^2}{2 \\mu_0 \\hat{U}_1}$","");
+         continue;
+      }
+      if(lowercase == "vg_db") {
+         outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_db",CellParams::D_B,1));
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\frac{\\abs{\\Delta B_1}}{\\hat{B}_1}$","");
+         continue;
+      }
```

Comment:
Hmm, these datareducers have a little bit inconsistent parameter names. In other places, we have "_derivatives" for derivative quantities, such as b_vol_derivatives.

Reply:
They're not exactly derivatives, just the maxima of the deltas of these variables to neighbouring cells scaled by their (or some other variables') maxima. Since they're still tied to the Vlasov grid, should I leave the prefix `vg_` or not? For example, should `vg_alpha` be renamed to `vg_amr_alpha` or `amr_alpha`?

Reply:
vg_ should be retained. vg_amr_alpha is probably best.

Reply:
Done, for all the new reducers including `vg_reflevel`